test/sequential/test-child-process-execsync.js

```diff
@@ -56,7 +54,7 @@ try {
 } finally {
   assert.strictEqual(ret, undefined,
                      `should not have a return value, received ${ret}`);
-  assert.strictEqual(caught, true, 'execSync should throw');
```

Comment:
This should not be changed. 1. caught should be a boolean and ok does a loose equal check. 2. The second argument would now be the error message and the third argument would be ignored.

Reply:
IMO, this should definitely be changed, but not the way it is in this PR right now.

There are two options.

Since `caught` is initialized to `false` and only ever changed to `true` inside of code that is in this file, `strictEqual()` is unnecessary. So this should be fine:

```js
assert.ok(caught, 'execSync should throw');
```

Alternatively, if you want to insist on `strictEqual()` for the type-checking aspect, that's fine but then the string literal third argument should be removed (and possibly moved to a comment above the call) because you care about whether the test failed because `caught` was `false` or `caught` was the string `"true"` instead of the boolean `true`, etc. 

```js
// execSync should throw
assert.strictEqual(caught, true);
```

I think either of these options are better than what's there now.

Reply:
Changed `assert.ok(caught, true, 'execSync should throw');`
 to `assert.ok(caught, 'execSync should throw');`